### ISSUE TITLE
Working with docker on Raspian?

### ISSUE THREAD

**Issue by milux** [ryansch/docker-unifi-rpi#13]
I have a "normal" docker installation on Raspian, can I run your containers with that environment as well?
The last container I tried was that one, but it crashes shortly after starting initialization:
https://hub.docker.com/r/erikdevries/rpi-unifi/
Thanks for all your work on this!

Best, Michael

**Comment by ryansch** [comment]
@milux Yeah this is just a docker container image.  I have getting started instructions for people who don't know how to get docker running on their pi but you can run containers from this image anywhere you have docker and arm.

**Comment by milux** [comment]
Thank you, @ryansch!
Actually, I answered the question for myself just before you posted your answer.
It works **great**, thanks man! 😄
I tried two other containers, but this one is definitely the best!
In case you're interested, this is the run command I used to start the container (detached, automatic start on host reboot):
`docker run --net=host --name unifi -d --restart unless-stopped ryansch/unifi-rpi:v5`

Just in case you're interested:
I used those commands to change the keystore in the docker containers to a self-signed certificated that I've imported into the machines I use. Not as good as a Let's Encrypt solution, but good enough for the moment:
```
docker cp unifi:/var/lib/unifi/keystore ~/docker_unifi_keystore_backup
docker cp ~/keystore unifi:/var/lib/unifi/keystore
```
I created this certificate using the following commands:
```
openssl pkcs12 -export -passout pass:aircontrolenterprise -in unifi.crt -inkey unifi.key -out unifi.p12 -name unifi
keytool -importkeystore -deststorepass aircontrolenterprise -destkeypass aircontrolenterprise -destkeystore keystore -srckeystore unifi.p12 -srcstoretype PKCS12 -srcstorepass aircontrolenterprise -alias unifi
```
